modules/introduction/pages/intro.adoc

```diff
@@ -1,3 +1,87 @@
+
+From the developer point of view, IDPs integrate into existing software development lifecycles (SDLC) workflows in an organization, which usually revolves around a distributed version control system like Git, and the IDP further augments it with automation. The developer has minimal to zero reliance on operations teams for rolling out software to production. They can request resources, spin up fully provisioned environments, deploy, rollback, and troubleshoot applications independently.
+
+== Platform Engineering
+
+The terms **Platform Engineering** and **Internal Developer Platform** are sometimes used interchangeably in the industry. For the purposes of this introductory course, you can consider them the same. Strictly speaking, IDPs are a major component of platform engineering, but platform engineering encompasses many more practices and tools. 
+
+The page at https://platformengineering.org/blog/what-is-platform-engineering created by the platform engineering community explains the details.
+
+== Why do you need an Internal Developer Platform (IDP)?
+
+The page at https://internaldeveloperplatform.org/why-build-an-internal-developer-platform provides a detailed explanation for why you need an IDP.
+
+Additionally, below is a short 6 minute introduction to Backstage and the rationale for building an Internal Developer Platform (IDP).
+
+video::n1IrNe5MmZg[youtube,title=Backstage.io - An Open Source Portal for Taming Developer Chaos,width=640,height=480]
+
+== Backstage, Project Janus and Red Hat Developer Hub
+
+image::backstage-janus-rhdh-flow.png[width=800px]
+
+=== Backstage
+
+**Backstage** is an open-source platform for building developer portals created by Spotify and donated to the Cloud Native Computing Foundation (CNCF). Rather than a product that is ready to use by developers, Backstage provides the core features for building developer portals, and has an extensive list of plugins supporting integration with platforms, runtimes and external APIs. Platform engineering teams take the base Backstage platform, and then extend and customize it for their developers' needs. Refer to the Backtage website for a full list of plugins.
+
+=== Project Janus
+
+**Project Janus** is a Red Hat-sponsored open source community for building IDPs and plugins using Backstage.
+It is the “mid-stream” for the Red Hat Developer Hub product, sitting between Backstage upstream and the supported Red Hat product offering which is focused on four areas:
+
+* Installation on OpenShift Container Platform
+* Developing and enhancing plugins, specifically targeting other Red Hat products like Red Hat OpenShift Pipelines, Red Hat OpenShift GitOps, Red Hat Single Sign-On (RHSSO) and many more
```

Comment:
From what I heard the RHSSO offering is either already discontinued or will be so very soon? Maybe it makes sense to consider this here?

Reply:
Replaced with Red Hat build of Keycloak